docs/concepts/guide-data-apis.md

```diff
@@ -53,12 +53,27 @@ Here's the full list of operators supported:
 
 | op  |  Meaning  | Supported types | Examples  |
 |-----|-----------|----------|----------------|
-| eq  | Equals     | integer, string | project_id=eq.pr123 |
+| eq  | Equals     | integer, string, timestamp | project_id=eq.pr123 |
+| neq  | Not Equals     | integer, string, timestamp | amount=neq.531 |
 | lt  | Less than  | integer, string, timestamp | amount=lt.100 |
 | lte | Less than or equal to  | integer, string, timestamp | updated_at=lte.2020-06-01T00:00:00.000-08:00 |
 | gt | Greater than  | integer, string, timestamp | amount=gt.100 |
 | gte | Greater than or equal to  | integer, string, timestamp| updated_at=gte.2020-06-01T00:00:00.000-08:00 |
-| in | Is one of  | integer, string | id=in.(id1,id2,id3) |
+| in | Is one of | integer, string | id=in.(id1,id2,id3) |
+| not_in | Is not one of | integer, string | id=not_in.(id1,id2,id3) |
+| is | Is equals to | null, boolean | is_reimbursable=is.true |
+| is_not | Is not equals to | null, boolean | is_reimbursable=is_not.true |
+| like | Matches case-sensitively with pattern  | integer, string | cost_center->name=like.%porate%, cost_center->name=like.Corporate |
+| not_like | Doesn't match case-sensitively with pattern  | integer, string | cost_center->name=not_like.%porate%, cost_center->name=not_like.Corporate |
+| ilike | Matches pattern while ignoring case sensitivity  | integer, string | creator_user_id=ilike.%Klnkmsvw9%, creator_user_id=ilike.usyKlnkmsvw9 |
+| not_ilike | Doesn't match pattern while ignoring case sensitivity  | integer, string | creator_user_id=not_ilike.%Klnkmsvw9%, creator_user_id=not_ilike.usyKlnkmSVw9 |
+| cs | Contains  | array, jsonb | category_ids=cs.[115260, 115257], cost_center=cs.{"code":"13597", "name": "Corporate", "id": 6632} |
+| csn | Contains or is Null  | restricted array, restricted jsonb | restricted_category_ids=csn.[115260, 115257], restricted_cost_center=csn.{"code":"13597", "name": "Corporate", "id": 6632} |
+| cd | Contained by  | array, jsonb | category_ids=cd.[115260, 115257], cost_center=cd.{"code":"13597", "name": "Corporate", "id": 6632} |
+| not_cd | Not contained by | array, jsonb | category_ids=not_cd.[115260, 115257], cost_center=not_cd.{"code":"13597", "name": "Corporate", "id": 6632}|
+| ov | Have elements in common | array | category_ids=ov.[115249, 115248] |
+| any | Any of the elements are equal to | array | any.115257, any."KM"/any.KM |
```

Comment:
these examples are not good.
why no column names are present here, like in other examples